### DISCUSSION TITLE
Corn not valid ingredient for Vegetable Cooking Oil

### DISCUSSION THREAD

**Discussion by PizzaPolice** [cataclysmbn/Cataclysm-BN#2067] [General]
# Is your feature request related to a problem? Please describe.
Vegetable oil cooking recipe does not allow use of corn

# Describe the solution you'd like
Add corn to the allowed ingredients for the JSON recipe for "vegetable cooking oil"

# Describe alternatives you've considered
None

# Additional context
Add any other context (such as mock-ups, proof of concepts or screenshots) about the feature request here.
![image](https://user-images.githubusercontent.com/12305731/163732594-b2877740-46e1-4cf1-b87b-a4bb98d739fd.png)
Notice lack of corn

**Comment by Lamandus** [comment]
as far as I am aware, it is not easy to make oil out of corn.
